i18n/qqq.json

```diff
@@ -14,27 +14,27 @@
 	"echo-category-title-requestssl-new-request": "Echo category title for 'requestssl-new-request'.",
 	"echo-category-title-requestssl-request-comment": "Echo category title for 'requestssl-request-comment'.",
 	"echo-category-title-requestssl-request-status-update": "Echo category title for 'requestssl-request-status-update'.",
-	"echo-pref-tooltip-requestssl-new-request": "Echo preference tooltip for 'requestssl-new-request'.",
-	"echo-pref-tooltip-requestssl-request-comment": "Echo preference tooltip for 'requestssl-request-comment'.",
-	"echo-pref-tooltip-requestssl-request-status-update": "Echo preference tooltip for 'requestssl-request-status-update'.",
+	"echo-pref-tooltip-requestssl-new-request": "Echo preference tooltip for 'requestSSL-new-request'.",
```

Comment:
why is the SSL capitalised here but not right above?

Reply:
because I missed this one when fixing my editor's find and replace mess ups.

Fixed in ebeb86d.